assets/src/edit-story/components/library/text/fontPreview.js

```diff
@@ -49,14 +49,21 @@ const Text = styled.span`
   color: ${({ theme }) => theme.colors.fg.v1};
 `;
 
-function FontPreview({ title, fontFamily, ...fontProps }) {
+function FontPreview({
+  title,
+  fontFamily,
+  content,
+  fontWeight,
+  fontStyle,
+  ...fontProps
+}) {
```

Comment:
Note that if you're taking all these out of `fontProps` then the `Text` below will not get the needed props for styling.